perses/app/fah_generator.py

```diff
@@ -315,29 +309,26 @@ def run(yaml_filename=None,index=None):
 
     import os
     # make master directories
-    if not os.path.exists('13406'):
-        os.makedirs('1340r/RUNS/')
-    if not os.path.exists('13407'):
-        os.makedirs('13407/RUNS/')
+    if not os.path.exists('13408'):
```

Comment:
@hannahbrucemacdonald Shouldn't these RUN indices be specified in the YAML file?